src/main/java/com/google/api/codegen/transformer/StaticLangResourceObjectTransformer.java

```diff
@@ -75,12 +75,13 @@ public RequestObjectParamView generateRequestObjectParam(
     String formatMethodName = null;
     if (context.getFeatureConfig().useResourceNameFormatOption(fieldConfig)) {
       if (fieldConfig.requiresParamTransformation()
-          && !fieldConfig.requiresParamTransformationFromAny()) {
+          && !fieldConfig.requiresParamTransformationFromAny()
+          && !featureConfig.useInheritanceForOneofs()) {
```

Comment:
Is it possible to include this in the `requiresParamTransformation` function somehow? I can't see how on a quick look, but wanted to check whether you had considered this. It would mean we wouldn't need the new feature config.

Reply:
The answer needs to be different for C# vs Java, so it has to be controlled in feature config (which is a language-specific knob).